packages/toolkit/package.json

```diff
@@ -38,12 +37,15 @@
     "css-loader": "^6.7.1",
     "cssnano": "^5.1.7",
     "eslint-webpack-plugin": "^3.1.1",
+    "fast-glob": "^3.2.11",
+    "fs": "^0.0.1-security",
     "html-webpack-plugin": "^5.5.0",
     "ignore-emit-webpack-plugin": "2.0.6",
     "image-minimizer-webpack-plugin": "^3.2.3",
     "jest": "^27.5.1",
     "mini-css-extract-plugin": "^2.6.0",
     "minimist": "^1.2.6",
+    "path": "^0.12.7",
```

Comment:
same, no need to install path.

Reply:
fixed in https://github.com/10up/10up-toolkit/pull/195/commits/6d906e6ced9c8e998a18b6f301f669f01c737009